packages/client/src/rpc/modules/eth.ts

```diff
@@ -28,7 +29,7 @@ import type { EthereumClient } from '../../index.js'
 import type { EthProtocol } from '../../net/protocol/index.js'
 import type { FullEthereumService, Service } from '../../service/index.js'
 import type { RpcTx } from '../types.js'
-import type { Block, JsonRpcBlock } from '@ethereumjs/block'
+import type { JsonRpcBlock } from '@ethereumjs/block'
```

Comment:
How come this is added but nothing is done with it in this PR? I don't see anything in the diff :thinking:

Reply:
It should have moved block from a type import to a regular import

Reply:
Oh right yes this makes sense, thanks

Reply:
(its actually in the diff :sweat_smile: - time to call it a day I guess :smile: )